src/ape/api/providers.py

```diff
@@ -931,6 +942,28 @@ def start(self, timeout: int = 20):
                     time.sleep(0.1)
                     _timeout.check()
 
+    def produce_stdout_queue(self):
```

Comment:
@johnson2427 let me know your thoughts on this madness! It seems to work.

Reply:
how much of a headache do you think it would be to make a test for this? Normally I wouldn't think it would be, but the required ties to hardhat/starknet/foundry make me wonder if it would be a giant headache

Reply:
If we normally do the hardhat test in these scenarios, let's keep it that way until we come up with a better way to do it